### ISSUE TITLE
symsorter fails when encountering .net files

### ISSUE THREAD

**Issue by kristjanvalur** [getsentry/symbolicator#416]
### Environment

Win64

### Steps to Reproduce

Point symsorter at a place where there are .NET binaries, such as the the `Engine\Binaries` under UnrealEngine4 installation.

### Expected Result
symsorter ignores files that it doesn't understand.

### Actual Result
symsorter fails and crashes with logs like this:
```
SwarmAgent.exe (exe, x86) -> d:\scratch\output2\Win64\a6/153c0e-3e1a-4667-83f9-621f6a4fc7c3/executable
thread '<unnamed>' panicked at 'thread 'called `Result::unwrap()` on an `Err` value: ParseDebugIdError<unnamed>', ' panicked at 'symsorter\src\app.rscalled `Result::unwrap()` on an `Err` value: ParseDebugIdError:', 139symsorter\src\app.rs:thread ':64139<unnamed>
' panicked at ':called `Result::unwrap()` on an `Err` value: ParseDebugIdErrorthread '64thread '', <unnamed>
<unnamed>symsorter\src\app.rs' panicked at '' panicked at 'thread ':called `Result::unwrap()` on an `Err` value: ParseDebugIdErrorcalled `Result::unwrap()` on an `Err` value: ParseDebugIdError<unnamed>139' panicked at '', thread '', :called `Result::unwrap()` on an `Err` value: ParseDebugIdErrorthread 'symsorter\src\app.rs<unnamed>symsorter\src\app.rs64', <unnamed>thread ':<unnamed>:
symsorter\src\app.rs' panicked at '' panicked at '139' panicked at '139:called `Result::unwrap()` on an `Err` value: ParseDebugIdErrorcalled `Result::unwrap()` on an `Err` value: ParseDebugIdError:called `Result::unwrap()` on an `Err` value: ParseDebugIdError:139', ', 64symsorter\src\app.rs64:thread '64',
:
```

**Comment by kristjanvalur** [comment]
Here is the file in question in case it helps
[SwarmAgent.zip](https://github.com/getsentry/symbolicator/files/6404104/SwarmAgent.zip)

**Comment by Swatinem** [comment]
@kristjanvalur sorry for the delay, and thanks for reporting this. As mentioned in #427 it would be good to pinpoint the exact commit you are using.
I downloaded the file you attached and can work with that correctly, although on mac. I see you are on windows.
Another thing I notice now is that you have the complete output for that file

```
SwarmAgent.exe (exe, x86) -> d:\scratch\output2\Win64\a6/153c0e-3e1a-4667-83f9-621f6a4fc7c3/executable
```

Which means that that file was processed correctly, and it is the *next* file that is causing this problem.

**Comment by kristjanvalur** [comment]
I checked again and it is fixed in d7e43cb9fbe1e92f8f7c3b3dc9bbed1329f7a7c7
Previously, I had been using the parent commit to this.

**Comment by Swatinem** [comment]
Ah that definitely makes sense, there was a `get_unified_id(&obj).parse().unwrap()` on that line.
